.prettierrc.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "semi": false,
+  "singleQuote": true
+}
```

Comment:
The idea is to leave the config as zero-config as possible and let the user customize it. Not sure if it's a good idea to add this in here.